launcher/src/EnsureInstanceRunning.scala

```diff
@@ -230,8 +230,14 @@ case class EnsureInstanceRunning(userName: String = os.proc("whoami").call().out
             .replace("$DEVBOX_USER", userName)
             .replace("$PYTHON_EXECUTABLE", pythonExecutable)
       ),
+      Left(
+        s"/home/$userName/.devbox/git-shim.py" ->
+          os.read(os.resource / "git-shim.py")
+      ),
       Right("systemctl enable devbox_daemon"),
       Right("systemctl start devbox_daemon"),
+      Right(s"chmod a+x /home/$userName/.devbox/git-shim.py"),
+      Right(s"""echo "alias git=/home/$userName/.devbox/git-shim.py" >> /home/$userName/.bash_aliases""")
```

Comment:
Could we make this whole feature (skipping .git, setting up git-shim, port forwarding, proxy server) enabled via a `--proxy-git-commands true` flag? We can pass that flag in by default in our launcher script, but making it a flag would make it easy for people to opt out in the off chance that this causes issues

Reply:
One thing to note is that this only runs once, when provisioning a new instance, not when reusing an existing one, even stopped.